Add compatibility with Gnome-Shell v49

hamen opened the pull request:
Hi, I had to upgrade to Ubuntu 25.10 and gnome-shell v49 broke the extension.
I stitched it at my best. I have never worked on this type of extensions. 
It's here if it can be useful to anybody ☮️

<img width="1922" height="938" alt="image" src="https://github.com/user-attachments/assets/34b4f9d4-a369-4f51-96af-35d94fff7140" />

FelipeFTN reviewed (commented):
**Hello buddy! Thank you for your contribution! Nice work!**

I think the only necessary change for your contribution to work (compatibility with Gnome-Shell v49), should be metadata.json...

Can you keep it simple for now and open another pull request with the polishings you did and explaining them with the right comments?

Also, have you tried to build and run the project just by running `make debug`?
You can take a look at **debug.md** it should also work to **wayland** too. 🎉 

So, please, keep only the metadata.json for this pull request and open a new one for the other changes, feature or improvements to the code! 🙏 

Also, remove the build-and-install.sh if the Makefile is working properly — at least if we don't have some reason for the redundancy! :100:

hamen commented:
Thank you for the feedback.

I’ve reduced this PR to the minimum and now only change metadata.json to add GNOME Shell 49 to shell-version.
I’ve removed the extra helper script and reverted all other files to match upstream.
I’ll open a separate PR with the code polishings and detailed explanations.

I think we can close this 🙏

FelipeFTN commented:
**Sounds perfect, thanks!** I will be reviewing and testing the #116 Pull Request and I'll update you soon! :rocket:
